_pages/en_US/include/rmproxy.txt

```diff
@@ -0,0 +1,10 @@
+In this section, you will restore the Internet settings back to normal.
+
+1. Launch the System Settings application
+1. Navigate to `Internet Settings` -> `Connection Settings`
+1. Click on your network connection slot and navigate to `Change Settings` -> `Next Page (right arrow)` -> `Proxy Settings`
+1. Set "Proxy Settings" to "No"
+1. Click OK, then click Save
+1. When prompted, click "Test" to perform the connection test
+	+ The test should succeed
+1. Power off your device
```

Comment:
this doesn't flow to the next page correctly. we power off the device here, but we then also power off in finalizing-setup

we need to pick one to drop (probably this one)

Reply:
this point is moot as all the pages do a mix of it